src/sass/components/_menu.scss

```diff
@@ -0,0 +1,86 @@
+.rvt-menu {
+  &__list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+  
+  &__item {
+    margin: 0;
+  }
+  
+  &__item a,
+  &__item button {
+    text-decoration: none;
+    display: block;
+    padding: $xxs $sm;
+    position: relative;
+    color: $color-black;
+    display: flex;
+    align-items: center;
+    -webkit-appearance: none;
+    border: none;
+  }
+  
+  &__item a[aria-current] {
+    background-color: $color-black--100;
+  }
+  
+  &__item a[aria-current]::after,
+  &__item a:hover::after,
+  &__item button:hover::after {
+    content: "";
+    display: block;
+    width: $xxs;
+    height: 100%;
+    background-color: $color-blue--600;
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+}
+
+@mixin horizontal-menu($selector) {
+  .rvt-menu {
+    #{$selector} {
+      border-bottom: 1px solid $color-black--200;
+    }
+
+    #{$selector} &__list {
+      display: flex;
+    }
+
+    #{$selector} &__item {
+      flex-shrink: 0;
+    }
+
+    #{$selector} &__item a,
+    #{$selector} &__item button {
+      padding: $sm $sm*1.25;
+    }
+    
+    #{$selector} &__item a[aria-current] {
+      background-color: transparent;
+    }
+    
+    #{$selector} &__item a[aria-current]::after,
+    #{$selector} &__item a:hover::after,
+    #{$selector} &__item button:hover::after {
+      width: 100%;
+      height: $xxs;
+      bottom: 0;
+      top: auto;
+    }
+    
+    #{$selector} &__item a:hover::after,
+    #{$selector} &__item button:hover::after {
+      background-color: $color-black--200;
+    }
+  }
+}
+
+@include horizontal-menu('&--horizontal');
+
+@include mq(bp(md)) {
```

Comment:
One thing here. A while back we added regular variables for all the breakpoints. I'd kinda like to move away from using this `bp()` function that gets values out of the main `$breakpoints` map and instead use the regular named variables. So in this case `$breakpoint-md`. Makes things a little more understandable, I think.

Reply:
Sounds good to me! Would you like for me to go through and replace all instances of the `bp()` function with the breakpoint variable reference as a separate branch for the next release?

Reply:
Yeah, if you have time that would be great. I know there are some straggler instances of it out there. 👍

Reply:
I'll file an issue and get on it, thanks!